docs/reference/airnode/latest/packages/deployer.md

```diff
@@ -118,6 +118,30 @@ cp config/secrets.env.example config/secrets.env
 ```
 -->
 
+## Credentials
```

Comment:
I confirmed that the GCP instructions and both AWS approaches work using the command: `npx @api3/airnode-deployer list`.